usersimcrs/simulator/agenda_based/interaction_model.py

```diff
@@ -16,8 +16,21 @@
 class InteractionModel:
     """Represents an interaction model."""
 
-    START_INTENT = Intent("DISCLOSE.NON-DISCLOSE")
-    STOP_INTENT = Intent("STOP")
+    # TODO: These should probably be loaded from the interaction model yaml file.
```

Comment:
Yes. A list of minimum intents should be defined and initialized in `__init__`.

Reply:
OK, this is made part of https://github.com/iai-group/UserSimCRS/issues/111 then.